src/routes/videoPlaybackProxy.ts

```diff
@@ -77,28 +77,38 @@ videoPlaybackProxy.get("/", async (c) => {
 
     const fetchClient = await getFetchClient(konfigStore);
 
-    let googlevideoResponse = await fetchClient.call(
-        undefined,
-        `https://${host}/videoplayback?${queryParams.toString()}`,
-        {
-            method: "POST",
-            body: new Uint8Array([0x78, 0]), // protobuf: { 15: 0 } (no idea what it means but this is what YouTube uses),
-            headers: headersToSend,
-        },
-    );
+    let location = `https://${host}/videoplayback?${queryParams.toString()}`;
 
-    if (googlevideoResponse.headers.has("location")) {
-        googlevideoResponse = await fetchClient.call(
+    // https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-p2-semantics-17#section-7.3
+    // A maximum of 5 redirections is defined in the note of the section 7.3
+    // of this RFC, that's why `i < 5`
+    for (let i = 0; i < 5; i++) {
+        const googlevideoResponse: Response = await fetchClient.call(
             undefined,
-            googlevideoResponse.headers.get("location") as string,
+            location,
             {
-                method: "POST",
-                body: new Uint8Array([0x78, 0]), // protobuf: { 15: 0 } (no idea what it means but this is what YouTube uses)
+                method: "HEAD",
```

Comment:
I think `redirect: "manual"` can just be passed here, otherwise we're altering every fetch call in the system to not follow redirects.